### PR TITLE
Medium maze b listenter

### PR DESCRIPTION
**Pull request by lettucewrap**
Pull request for Mode buttons

### CODE REVIEW

**Review by cammurriel — APPROVED** [review]
good to go!

File: src/main/java/version1/gameUtil/settings/UIConfigurations.java [hunk 1]
```diff
@@ -159,6 +159,7 @@ public static void configure(ModeScreen modeScreen){
                 .createHardButton(hardButtonText)
                 .createGoToMenuButton(goToMenuText)
                 .createGoToMediumMazeButton(mediumButtonText)
+                .createGoToHardMazeButton(hardButtonText)
```
Comment (BenMob) [inline src/main/java/version1/gameUtil/settings/UIConfigurations.java]: Hi @cammurriel I see you have created a `createGoToHardMazeButton()` on line 162, and it seems to me like it is doing the same thing that the createHardButton() is doing on line 159.

I think it was my bad for not telling you that the methods were already created and that you just needed to fill them in.

File: src/main/java/version1/gameUtil/settings/UIConfigurations.java [hunk 2]
```diff
@@ -158,6 +158,7 @@ public static void configure(ModeScreen modeScreen){
                 .createMediumButton(mediumButtonText)
                 .createHardButton(hardButtonText)
                 .createGoToMenuButton(goToMenuText)
+                .createGoToMediumMazeButton(mediumButtonText)
```
Comment (BenMob) [inline src/main/java/version1/gameUtil/settings/UIConfigurations.java:161]: Lines 158 and 161 seem to be doing the same task.  You did not have to create new methods, you just had to fill in the ones that were already there. 

It is my bad, I should have mentioned that yesterday.  So we will need to remove one of them and keep the other. Same thing for all the other screens.

**Review by BenMob — COMMENTED** [review]
There seem to be duplicate methods and buttons here and there. We just need to fix that and we should be good to go

**Pull request merged**